ballerina/build.gradle

```diff
@@ -1326,6 +1330,84 @@ task testStdlibs() {
+    def jBallerinaPack = "${project.rootDir}/ballerina/build/target/extracted-distributions/jballerina-tools-zip"
+    def baseBuildPath = "${project.rootDir}/ballerina/build/bbe-pack"
+    def distPath = "${baseBuildPath}/jballerina-tools-${ballerinaVersion}"
+    def devtoolsTestList = []
+
+    PatternSet patternSet = new PatternSet();
+    patternSet.exclude("**/.ballerina/**");
+    patternSet.exclude("**/Ballerina.toml");
+    patternSet.exclude("**/Ballerina.lock");
+    patternSet.exclude("**/ballerina-internal.log")
+    def ignoreList = []
+
+    inputs.files(files("${distPath}/devtools-integration-tests").asFileTree.matching(patternSet))
+
+    doFirst {
+        // Copy compiled pack to baseBuild temp directory
+        copy {
+            from jBallerinaPack
+            into baseBuildPath
+        }
+
+        //Initialize Ballerina project
+        exec {
+             if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+                 workingDir "${baseBuildPath}"
+                 commandLine 'cmd', '/c', "jballerina-tools-${ballerinaVersion}/bin/ballerina.bat", 'new', 'DevToolsTestProject'
+             } else {
+                 workingDir "${baseBuildPath}"
+                 commandLine 'sh', '-c', "jballerina-tools-${ballerinaVersion}/bin/ballerina new DevToolsTestProject"
+             }
+        }
+        
+        // Copy modules
+        def src = "${project.rootDir}/devtools-integration-tests/"
+        def dis = "${baseBuildPath}/DevToolsTestProject/src/"
+        copy {
+            from(src)
+            into dis
+        }
+
+        def inputFile = new File("${baseBuildPath}/DevToolsTestProject/src/index.json")
+        def devtoolsTests = new JsonSlurper().parseText(inputFile.text)
+        
+        devtoolsTests.each { devtoolsTest ->
+            def folder = new File("${baseBuildPath}/DevToolsTestProject/src/${devtoolsTest.path}")
+            if (folder.exists()) {
+                devtoolsTestList.push("$devtoolsTest.path")
+            }
+        }
+    }
+
+    doLast {
+        ignoreList.each { String elements ->
+            devtoolsTestList.remove("$elements")
+        }
+
+        devtoolsTestList.each { String devtoolsTest ->
+            def additionalBuildParams = ""
+            if (devtoolsTest == "config-api-test") {
+                additionalBuildParams = "--user.name=waruna"
+            }
+
+            if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+                 exec {
+                     workingDir "${buildDir}/DevToolsTestProject"
+                     commandLine 'cmd', '/c', "${distPath}/bin/ballerina.bat build -c --experimental ${devtoolsTest} --b7a.home=${distPath} ${additionalBuildParams}"
+                 }
+             } else {
+                 exec {
+                     workingDir "${baseBuildPath}/DevToolsTestProject"
+                     commandLine 'sh', '-c', "${distPath}/bin/ballerina build -c --experimental ${devtoolsTest} --b7a.home=${distPath} ${additionalBuildParams}"
```

Comment:
Do we need the "--experimental" flag?
AFAIK we are not using any experimental features.

Reply:
Removed unnecessary experimental tag